gempy/core/data_modules/geometric_data.py

```diff
@@ -951,7 +951,7 @@ def plane_fit(point_list):
 
 
 @_setdoc_pro([SurfacePoints.__doc__, Orientations.__doc__, Grid.__doc__])
-class RescaledData(object):
+class ScalingSystem(object):
```

Comment:
I think PreProcessing or Transformation or Standardization would be a better name for the class, as this is really just a data transformation and you could incorporate other kinds of anisotropy corrections in the future see https://scikit-learn.org/stable/modules/preprocessing.html#standardization-or-mean-removal-and-variance-scaling

Reply:
In the long term I think it is a good idea to give it a more general name (maybe Scaling is a subset of PreProcessing). So far I am going to leave Scaling because the default behavior is just the re-scaling to value between 0-1. But I keep in mind your suggestion for the next refactor